airflow/decorators/__init__.pyi

```diff
@@ -239,5 +241,11 @@ class TaskDecoratorCollection:
         :param cap_add: Include container capabilities
         """
         # [END decorator_signature]
+    @overload
+    def sensor(self, python_callable: Optional[Callable] = None, **kwargs) -> TaskDecorator:
```

Comment:
This is meaningless. Please see above for examples on how to write a useful type annotation.

Reply:
Can you be more specific?

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/howto/create-custom-decorator.html#optional-adding-ide-auto-completion-support

Reply:
Will fix!